### PR TITLE
I have implemented a function (end auction).

### PR DESCRIPTION
**Pull request by muskankapoor**
I have implemented a function (end auction).
Please approve it.

### CODE REVIEW

**Review by xiaoxiazhang00 — APPROVED** [review]

**Comment by xiaoxiazhang00** [comment]
please make sure your codes work properly before you merge.

File: contracts/Auction.sol [hunk 1]
```diff
@@ -113,7 +113,12 @@ contract Auction {
 
 
     function endAuction() public {
+        require(now >=  end_time; "Auction not yet ended.");
+        require(!ended, "auction  has already ended.");
 
+ 
+        ended = true;
+        bidder_address.transfer(highest_bid);
```
Comment (xiaoxiazhang00) [inline contracts/Auction.sol]: do we have `highest_bid ` member ? `current_highest_bidder` ?

**Review by xiaoxiazhang00 — CHANGES_REQUESTED** [review]
please make sure the code works properly before submit it to merge

File: contracts/Auction.sol (hunk 1 above)
Comment (muskankapoor) [reply contracts/Auction.sol]: Yes, they are. You can also check. one is in the struct. 
and another is defined also.

**Review by xiaoxiazhang00 — APPROVED** [review]

**Pull request merged**